Sources/MongoSwift/Operations/CreateIndexesOperation.swift

```diff
@@ -0,0 +1,56 @@
+import mongoc
+
+/// Options to use when creating a new index on a `MongoCollection`.
+public struct CreateIndexOptions: Encodable {
+    /// An optional `WriteConcern` to use for the command
+    public let writeConcern: WriteConcern?
+
+    /// Initializer allowing any/all parameters to be omitted.
+    public init(writeConcern: WriteConcern? = nil) {
+        self.writeConcern = writeConcern
+    }
+}
+
+/// An operation corresponding to a "createIndexes" command.
+internal struct CreateIndexesOperation<T: Codable>: Operation {
+    private let collection: MongoCollection<T>
+    private let models: [IndexModel]
+    private let options: CreateIndexOptions?
+
+    internal init(collection: MongoCollection<T>,
+                  models: [IndexModel],
+                  options: CreateIndexOptions?) {
+        self.collection = collection
+        self.models = models
+        self.options = options
+    }
+
+    internal func execute() throws -> [String] {
+        let collName = String(cString: mongoc_collection_get_name(self.collection._collection))
```

Comment:
why aren't we using the [name property](https://github.com/mongodb/mongo-swift-driver/blob/master/Sources/MongoSwift/MongoCollection.swift#L30)?

Reply:
great point, updated.